### ISSUE TITLE
Multiqc not showing all Star 2.7.1a results

### ISSUE THREAD

**Issue by agroppi** [ewels/MultiQC#987]
**Description of bug:**
I used Multiqc v1.7 to compile results from STAR 2.7.1a.
I have 30 samples. 
The multiqc report show the 30 General Statistics but for the STAR Alignment scores and Gene counts Multiqc show only the plots for 15 samples ... 

**MultiQC Error log:**
```
[2019-07-04 17:58:38,374] multiqc                                            [DEBUG  ]  No MultiQC config found: /gpfs/home/ag/multiqc/lib/python3.6/site-packages/multiqc_config.yaml
[2019-07-04 17:58:38,375] multiqc                                            [DEBUG  ]  No MultiQC config found: /home/ag/.multiqc_config.yaml
[2019-07-04 17:58:38,375] multiqc                                            [DEBUG  ]  No MultiQC config found: multiqc_config.yaml
[2019-07-04 17:58:38,375] multiqc                                            [DEBUG  ]  Command used: /home/ag/multiqc/bin/multiqc .
[2019-07-04 17:58:38,629] multiqc                                            [DEBUG  ]  Latest MultiQC version is v1.7
[2019-07-04 17:58:38,629] multiqc                                            [INFO   ]  This is MultiQC v1.7
[2019-07-04 17:58:38,629] multiqc                                            [DEBUG  ]  Command     : /home/ag/multiqc/bin/multiqc .
[2019-07-04 17:58:38,629] multiqc                                            [DEBUG  ]  Working dir : /gpfs/scratch/ag/RNAStar/STAR_Mapping/multiqc_input_files
[2019-07-04 17:58:38,629] multiqc                                            [INFO   ]  Template    : default
[2019-07-04 17:58:38,629] multiqc                                            [INFO   ]  Searching '.'
[2019-07-04 17:58:38,630] multiqc                                            [DEBUG  ]  Analysing modules: custom_content, conpair, peddy, methylQA, phantompeakqualtools, qualimap, preseq, quast, qorts, rna_seqc, rsem, rseqc, busco, goleft_indexcov, disambiguate, supernova, deeptools, sargasso, verifybamid, mirtrace, happy, homer, macs2, theta2, snpeff, gatk, htseq, bcftools, featureCounts, dedup, damageprofiler, picard, prokka, samblaster, samtools, bamtools, jellyfish, vcftools, longranger, stacks, bbmap, bismark, biscuit, hicexplorer, hicup, hicpro, salmon, kallisto, slamdunk, star, hisat2, tophat, bowtie2, bowtie1, kat, leehom, adapterRemoval, clipandmerge, cutadapt, flexbar, trimmomatic, skewer, sortmerna, biobloomtools, fastq_screen, afterqc, fastp, fastqc, minionqc, clusterflow, bcl2fastq, interop, flash
[2019-07-04 17:58:38,630] multiqc                                            [DEBUG  ]  Using temporary directory for creating report: /tmp/tmpdbl6vtcu
[2019-07-04 17:58:44,178] multiqc                                            [DEBUG  ]  No samples found: custom_content
[2019-07-04 17:58:44,189] multiqc                                            [DEBUG  ]  No samples found: conpair
[2019-07-04 17:58:44,214] multiqc                                            [DEBUG  ]  No samples found: peddy
[2019-07-04 17:58:44,230] multiqc                                            [DEBUG  ]  No samples found: methylQA
[2019-07-04 17:58:44,238] multiqc                                            [DEBUG  ]  No samples found: phantompeakqualtools
[2019-07-04 17:58:44,263] multiqc                                            [DEBUG  ]  No samples found: qualimap
[2019-07-04 17:58:44,276] multiqc                                            [DEBUG  ]  No samples found: preseq
[2019-07-04 17:58:44,286] multiqc                                            [DEBUG  ]  No samples found: quast
[2019-07-04 17:58:44,318] multiqc                                            [DEBUG  ]  No samples found: qorts
[2019-07-04 17:58:44,327] multiqc                                            [DEBUG  ]  No samples found: rna_seqc
[2019-07-04 17:58:44,340] multiqc                                            [DEBUG  ]  No samples found: rsem
[2019-07-04 17:58:44,361] multiqc                                            [DEBUG  ]  No samples found: rseqc
[2019-07-04 17:58:44,369] multiqc                                            [DEBUG  ]  No samples found: busco
[2019-07-04 17:58:44,394] multiqc                                            [DEBUG  ]  No samples found: goleft_indexcov
[2019-07-04 17:58:44,403] multiqc                                            [DEBUG  ]  No samples found: disambiguate
[2019-07-04 17:58:44,411] multiqc                                            [DEBUG  ]  No samples found: supernova
[2019-07-04 17:58:44,434] multiqc                                            [DEBUG  ]  No samples found: deeptools
[2019-07-04 17:58:44,445] multiqc                                            [DEBUG  ]  No samples found: sargasso
[2019-07-04 17:58:44,456] multiqc                                            [DEBUG  ]  No samples found: verifybamid
[2019-07-04 17:58:44,464] multiqc                                            [DEBUG  ]  No samples found: mirtrace
[2019-07-04 17:58:44,474] multiqc                                            [DEBUG  ]  No samples found: happy
[2019-07-04 17:58:44,486] multiqc                                            [DEBUG  ]  No samples found: homer
[2019-07-04 17:58:44,504] multiqc                                            [DEBUG  ]  No samples found: macs2
[2019-07-04 17:58:44,512] multiqc                                            [DEBUG  ]  No samples found: theta2
[2019-07-04 17:58:44,532] multiqc                                            [DEBUG  ]  No samples found: snpeff
[2019-07-04 17:58:44,561] multiqc                                            [DEBUG  ]  No samples found: gatk
[2019-07-04 17:58:44,568] multiqc                                            [DEBUG  ]  No samples found: htseq
[2019-07-04 17:58:44,577] multiqc                                            [DEBUG  ]  No samples found: bcftools
[2019-07-04 17:58:44,584] multiqc                                            [DEBUG  ]  No samples found: featureCounts
[2019-07-04 17:58:44,592] multiqc                                            [DEBUG  ]  No samples found: dedup
[2019-07-04 17:58:44,599] multiqc                                            [DEBUG  ]  No samples found: damageprofiler
[2019-07-04 17:58:44,640] multiqc                                            [DEBUG  ]  No samples found: picard
[2019-07-04 17:58:44,659] multiqc                                            [DEBUG  ]  No samples found: prokka
[2019-07-04 17:58:44,668] multiqc                                            [DEBUG  ]  No samples found: samblaster
[2019-07-04 17:58:44,685] multiqc                                            [DEBUG  ]  No samples found: samtools
[2019-07-04 17:58:44,695] multiqc                                            [DEBUG  ]  No samples found: bamtools
[2019-07-04 17:58:44,703] multiqc                                            [DEBUG  ]  No samples found: jellyfish
[2019-07-04 17:58:44,721] multiqc                                            [DEBUG  ]  No samples found: vcftools
[2019-07-04 17:58:44,763] multiqc                                            [DEBUG  ]  No samples found: longranger
[2019-07-04 17:58:44,773] multiqc                                            [DEBUG  ]  No samples found: stacks
[2019-07-04 17:58:44,796] multiqc                                            [DEBUG  ]  No samples found: bbmap
[2019-07-04 17:58:44,820] multiqc                                            [DEBUG  ]  No samples found: bismark
[2019-07-04 17:58:44,831] multiqc                                            [DEBUG  ]  No samples found: biscuit
[2019-07-04 17:58:44,840] multiqc                                            [DEBUG  ]  No samples found: hicexplorer
[2019-07-04 17:58:44,851] multiqc                                            [DEBUG  ]  No samples found: hicup
[2019-07-04 17:58:44,862] multiqc                                            [DEBUG  ]  No samples found: hicpro
[2019-07-04 17:58:44,872] multiqc                                            [DEBUG  ]  No samples found: salmon
[2019-07-04 17:58:44,881] multiqc                                            [DEBUG  ]  No samples found: kallisto
[2019-07-04 17:58:44,899] multiqc                                            [DEBUG  ]  No samples found: slamdunk
[2019-07-04 17:58:45,049] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-CTRL-M17-T-AMB
[2019-07-04 17:58:45,139] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-CTRL-PRE-ENCAPS
[2019-07-04 17:58:45,228] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-INDOX-PRE-ENCAPS
[2019-07-04 17:58:45,377] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-CTRL-POST-ENCAPS
[2019-07-04 17:58:45,482] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-INDOX-CHAUF-4J
[2019-07-04 17:58:45,590] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-INDOX-M17-T-AMB
[2019-07-04 17:58:45,698] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-CTRL-CHAUFFAGE-4J
[2019-07-04 17:58:45,787] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-INDOX-POST-ENCAPS
[2019-07-04 17:58:45,897] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-CTRL-CHAUFFAGE-8H
[2019-07-04 17:58:45,988] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-INDOX-PRE-ENCAPS
[2019-07-04 17:58:46,074] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-INDOX-CHAUF-4J
[2019-07-04 17:58:46,162] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-INDOX-CHAUF-4J
[2019-07-04 17:58:46,314] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-INDOX-POST-ENCAPS
[2019-07-04 17:58:46,427] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-CTRL-PRE-ENCAPS
[2019-07-04 17:58:46,512] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-INDOX-M17-T-AMB
[2019-07-04 17:58:46,615] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-CTRL-POST-ENCAPS
[2019-07-04 17:58:46,718] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-INDOX-PRE-ENCAPS
[2019-07-04 17:58:46,821] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-CTRL-M17-T-AMB
[2019-07-04 17:58:46,910] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-INDOX-M17-T-AMB
[2019-07-04 17:58:46,994] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-INDOX-POST-ENCAPS
[2019-07-04 17:58:47,078] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-CTRL-CHAUFFAGE-8H
[2019-07-04 17:58:47,182] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-CTRL-CHAUFFAGE-4J
[2019-07-04 17:58:47,266] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-CTRL-CHAUFFAGE-8H
[2019-07-04 17:58:47,354] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-INDOX-CHAUF-8H
[2019-07-04 17:58:47,443] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-INDOX-CHAUF-8H
[2019-07-04 17:58:47,532] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-CTRL-POST-ENCAPS
[2019-07-04 17:58:47,617] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-CTRL-PRE-ENCAPS
[2019-07-04 17:58:47,721] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-CTRL-M17-T-AMB
[2019-07-04 17:58:47,805] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P3-INDOX-CHAUF-8H
[2019-07-04 17:58:47,911] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-CTRL-CHAUFFAGE-4J
[2019-07-04 17:58:47,911] multiqc.modules.star.star                          [INFO   ]  Found 30 reports and 30 gene count files
[2019-07-04 17:58:47,934] multiqc                                            [DEBUG  ]  No samples found: hisat2
[2019-07-04 17:58:47,943] multiqc                                            [DEBUG  ]  No samples found: tophat
[2019-07-04 17:58:47,954] multiqc                                            [DEBUG  ]  No samples found: bowtie2
[2019-07-04 17:58:47,964] multiqc                                            [DEBUG  ]  No samples found: bowtie1
[2019-07-04 17:58:47,973] multiqc                                            [DEBUG  ]  No samples found: kat
[2019-07-04 17:58:47,980] multiqc                                            [DEBUG  ]  No samples found: leehom
[2019-07-04 17:58:47,986] multiqc                                            [DEBUG  ]  No samples found: adapterRemoval
[2019-07-04 17:58:47,993] multiqc                                            [DEBUG  ]  No samples found: clipandmerge
[2019-07-04 17:58:47,999] multiqc                                            [DEBUG  ]  No samples found: cutadapt
[2019-07-04 17:58:48,005] multiqc                                            [DEBUG  ]  No samples found: flexbar
[2019-07-04 17:58:48,012] multiqc                                            [DEBUG  ]  No samples found: trimmomatic
[2019-07-04 17:58:48,031] multiqc                                            [DEBUG  ]  No samples found: skewer
[2019-07-04 17:58:48,038] multiqc                                            [DEBUG  ]  No samples found: sortmerna
[2019-07-04 17:58:48,045] multiqc                                            [DEBUG  ]  No samples found: biobloomtools
[2019-07-04 17:58:48,067] multiqc                                            [DEBUG  ]  No samples found: fastq_screen
[2019-07-04 17:58:48,074] multiqc                                            [DEBUG  ]  No samples found: afterqc
[2019-07-04 17:58:48,083] multiqc                                            [DEBUG  ]  No samples found: fastp
[2019-07-04 17:58:48,091] multiqc                                            [DEBUG  ]  No samples found: fastqc
[2019-07-04 17:58:48,097] multiqc                                            [DEBUG  ]  No samples found: minionqc
[2019-07-04 17:58:48,104] multiqc                                            [DEBUG  ]  No samples found: clusterflow
[2019-07-04 17:58:48,111] multiqc                                            [DEBUG  ]  No samples found: bcl2fastq
[2019-07-04 17:58:48,117] multiqc                                            [DEBUG  ]  No samples found: interop
[2019-07-04 17:58:48,124] multiqc                                            [DEBUG  ]  No samples found: flash
[2019-07-04 17:58:48,139] multiqc                                            [INFO   ]  Compressing plot data
[2019-07-04 17:58:48,160] multiqc                                            [INFO   ]  Report      : multiqc_report.html
[2019-07-04 17:58:48,160] multiqc                                            [INFO   ]  Data        : multiqc_data
[2019-07-04 17:58:48,161] multiqc                                            [DEBUG  ]  Moving data file from '/tmp/tmpdbl6vtcu/multiqc_data/multiqc_star.txt' to '/gpfs/scratch/ag/RNAStar/STAR_Mapping/multiqc_input_files/multiqc_data'
[2019-07-04 17:58:48,162] multiqc                                            [DEBUG  ]  Moving data file from '/tmp/tmpdbl6vtcu/multiqc_data/multiqc_general_stats.txt' to '/gpfs/scratch/ag/RNAStar/STAR_Mapping/multiqc_input_files/multiqc_data'
[2019-07-04 17:58:48,163] multiqc                                            [DEBUG  ]  Moving data file from '/tmp/tmpdbl6vtcu/multiqc_data/multiqc_sources.txt' to '/gpfs/scratch/ag/RNAStar/STAR_Mapping/multiqc_input_files/multiqc_data'
[2019-07-04 17:58:48,164] multiqc                                            [DEBUG  ]  Moving data file from '/tmp/tmpdbl6vtcu/multiqc_data/multiqc_data.json' to '/gpfs/scratch/ag/RNAStar/STAR_Mapping/multiqc_input_files/multiqc_data'
[2019-07-04 17:58:48,450] multiqc                                            [INFO   ]  MultiQC complete

```

**File that triggers the error:**
Multiqc report : 
[STAR_RNAStar_mapping_multiqc_report.zip](https://github.com/ewels/MultiQC/files/3361872/STAR_RNAStar_mapping_multiqc_report.zip)

**MultiQC run details (please complete the following):**
 - Command used to run MultiQC: /home/ag/multiqc/bin/multiqc .
 - MultiQC Version: MultiQC v1.7
 - Operating System: CentOS7
 - Python Version: 3.6.8
 - Method of MultiQC installation: conda

 **Additional context**
Listing of the directory submitted to Multiqc : 
```
P1-CTRL-CHAUFFAGE-4J_Log.final.out
P1-CTRL-CHAUFFAGE-4J_Log.out
P1-CTRL-CHAUFFAGE-4J_Log.progress.out
P1-CTRL-CHAUFFAGE-4J_ReadsPerGene.out.tab
P1-CTRL-CHAUFFAGE-4J_SJ.out.tab
P1-CTRL-CHAUFFAGE-8H_Log.final.out
P1-CTRL-CHAUFFAGE-8H_Log.out
P1-CTRL-CHAUFFAGE-8H_Log.progress.out
P1-CTRL-CHAUFFAGE-8H_ReadsPerGene.out.tab
P1-CTRL-CHAUFFAGE-8H_SJ.out.tab
P1-CTRL-M17-T-AMB_Log.final.out
P1-CTRL-M17-T-AMB_Log.out
P1-CTRL-M17-T-AMB_Log.progress.out
P1-CTRL-M17-T-AMB_ReadsPerGene.out.tab
P1-CTRL-M17-T-AMB_SJ.out.tab
P1-CTRL-POST-ENCAPS_Log.final.out
P1-CTRL-POST-ENCAPS_Log.out
P1-CTRL-POST-ENCAPS_Log.progress.out
P1-CTRL-POST-ENCAPS_ReadsPerGene.out.tab
P1-CTRL-POST-ENCAPS_SJ.out.tab
P1-CTRL-PRE-ENCAPS_Log.final.out
P1-CTRL-PRE-ENCAPS_Log.out
P1-CTRL-PRE-ENCAPS_Log.progress.out
P1-CTRL-PRE-ENCAPS_ReadsPerGene.out.tab
P1-CTRL-PRE-ENCAPS_SJ.out.tab
P1-INDOX-CHAUF-4J_Log.final.out
P1-INDOX-CHAUF-4J_Log.out
P1-INDOX-CHAUF-4J_Log.progress.out
P1-INDOX-CHAUF-4J_ReadsPerGene.out.tab
P1-INDOX-CHAUF-4J_SJ.out.tab
P1-INDOX-CHAUF-8H_Log.final.out
P1-INDOX-CHAUF-8H_Log.out
P1-INDOX-CHAUF-8H_Log.progress.out
P1-INDOX-CHAUF-8H_ReadsPerGene.out.tab
P1-INDOX-CHAUF-8H_SJ.out.tab
P1-INDOX-M17-T-AMB_Log.final.out
P1-INDOX-M17-T-AMB_Log.out
P1-INDOX-M17-T-AMB_Log.progress.out
P1-INDOX-M17-T-AMB_ReadsPerGene.out.tab
P1-INDOX-M17-T-AMB_SJ.out.tab
P1-INDOX-POST-ENCAPS_Log.final.out
P1-INDOX-POST-ENCAPS_Log.out
P1-INDOX-POST-ENCAPS_Log.progress.out
P1-INDOX-POST-ENCAPS_ReadsPerGene.out.tab
P1-INDOX-POST-ENCAPS_SJ.out.tab
P1-INDOX-PRE-ENCAPS_Log.final.out
P1-INDOX-PRE-ENCAPS_Log.out
P1-INDOX-PRE-ENCAPS_Log.progress.out
P1-INDOX-PRE-ENCAPS_ReadsPerGene.out.tab
P1-INDOX-PRE-ENCAPS_SJ.out.tab
P2-CTRL-CHAUFFAGE-4J_Log.final.out
P2-CTRL-CHAUFFAGE-4J_Log.out
P2-CTRL-CHAUFFAGE-4J_Log.progress.out
P2-CTRL-CHAUFFAGE-4J_ReadsPerGene.out.tab
P2-CTRL-CHAUFFAGE-4J_SJ.out.tab
P2-CTRL-CHAUFFAGE-8H_Log.final.out
P2-CTRL-CHAUFFAGE-8H_Log.out
P2-CTRL-CHAUFFAGE-8H_Log.progress.out
P2-CTRL-CHAUFFAGE-8H_ReadsPerGene.out.tab
P2-CTRL-CHAUFFAGE-8H_SJ.out.tab
P2-CTRL-M17-T-AMB_Log.final.out
P2-CTRL-M17-T-AMB_Log.out
P2-CTRL-M17-T-AMB_Log.progress.out
P2-CTRL-M17-T-AMB_ReadsPerGene.out.tab
P2-CTRL-M17-T-AMB_SJ.out.tab
P2-CTRL-POST-ENCAPS_Log.final.out
P2-CTRL-POST-ENCAPS_Log.out
P2-CTRL-POST-ENCAPS_Log.progress.out
P2-CTRL-POST-ENCAPS_ReadsPerGene.out.tab
P2-CTRL-POST-ENCAPS_SJ.out.tab
P2-CTRL-PRE-ENCAPS_Log.final.out
P2-CTRL-PRE-ENCAPS_Log.out
P2-CTRL-PRE-ENCAPS_Log.progress.out
P2-CTRL-PRE-ENCAPS_ReadsPerGene.out.tab
P2-CTRL-PRE-ENCAPS_SJ.out.tab
P2-INDOX-CHAUF-4J_Log.final.out
P2-INDOX-CHAUF-4J_Log.out
P2-INDOX-CHAUF-4J_Log.progress.out
P2-INDOX-CHAUF-4J_ReadsPerGene.out.tab
P2-INDOX-CHAUF-4J_SJ.out.tab
P2-INDOX-CHAUF-8H_Log.final.out
P2-INDOX-CHAUF-8H_Log.out
P2-INDOX-CHAUF-8H_Log.progress.out
P2-INDOX-CHAUF-8H_ReadsPerGene.out.tab
P2-INDOX-CHAUF-8H_SJ.out.tab
P2-INDOX-M17-T-AMB_Log.final.out
P2-INDOX-M17-T-AMB_Log.out
P2-INDOX-M17-T-AMB_Log.progress.out
P2-INDOX-M17-T-AMB_ReadsPerGene.out.tab
P2-INDOX-M17-T-AMB_SJ.out.tab
P2-INDOX-POST-ENCAPS_Log.final.out
P2-INDOX-POST-ENCAPS_Log.out
P2-INDOX-POST-ENCAPS_Log.progress.out
P2-INDOX-POST-ENCAPS_ReadsPerGene.out.tab
P2-INDOX-POST-ENCAPS_SJ.out.tab
P2-INDOX-PRE-ENCAPS_Log.final.out
P2-INDOX-PRE-ENCAPS_Log.out
P2-INDOX-PRE-ENCAPS_Log.progress.out
P2-INDOX-PRE-ENCAPS_ReadsPerGene.out.tab
P2-INDOX-PRE-ENCAPS_SJ.out.tab
P3-CTRL-CHAUFFAGE-4J_Log.final.out
P3-CTRL-CHAUFFAGE-4J_Log.out
P3-CTRL-CHAUFFAGE-4J_Log.progress.out
P3-CTRL-CHAUFFAGE-4J_ReadsPerGene.out.tab
P3-CTRL-CHAUFFAGE-4J_SJ.out.tab
P3-CTRL-CHAUFFAGE-8H_Log.final.out
P3-CTRL-CHAUFFAGE-8H_Log.out
P3-CTRL-CHAUFFAGE-8H_Log.progress.out
P3-CTRL-CHAUFFAGE-8H_ReadsPerGene.out.tab
P3-CTRL-CHAUFFAGE-8H_SJ.out.tab
P3-CTRL-M17-T-AMB_Log.final.out
P3-CTRL-M17-T-AMB_Log.out
P3-CTRL-M17-T-AMB_Log.progress.out
P3-CTRL-M17-T-AMB_ReadsPerGene.out.tab
P3-CTRL-M17-T-AMB_SJ.out.tab
P3-CTRL-POST-ENCAPS_Log.final.out
P3-CTRL-POST-ENCAPS_Log.out
P3-CTRL-POST-ENCAPS_Log.progress.out
P3-CTRL-POST-ENCAPS_ReadsPerGene.out.tab
P3-CTRL-POST-ENCAPS_SJ.out.tab
P3-CTRL-PRE-ENCAPS_Log.final.out
P3-CTRL-PRE-ENCAPS_Log.out
P3-CTRL-PRE-ENCAPS_Log.progress.out
P3-CTRL-PRE-ENCAPS_ReadsPerGene.out.tab
P3-CTRL-PRE-ENCAPS_SJ.out.tab
P3-INDOX-CHAUF-4J_Log.final.out
P3-INDOX-CHAUF-4J_Log.out
P3-INDOX-CHAUF-4J_Log.progress.out
P3-INDOX-CHAUF-4J_ReadsPerGene.out.tab
P3-INDOX-CHAUF-4J_SJ.out.tab
P3-INDOX-CHAUF-8H_Log.final.out
P3-INDOX-CHAUF-8H_Log.out
P3-INDOX-CHAUF-8H_Log.progress.out
P3-INDOX-CHAUF-8H_ReadsPerGene.out.tab
P3-INDOX-CHAUF-8H_SJ.out.tab
P3-INDOX-M17-T-AMB_Log.final.out
P3-INDOX-M17-T-AMB_Log.out
P3-INDOX-M17-T-AMB_Log.progress.out
P3-INDOX-M17-T-AMB_ReadsPerGene.out.tab
P3-INDOX-M17-T-AMB_SJ.out.tab
P3-INDOX-POST-ENCAPS_Log.final.out
P3-INDOX-POST-ENCAPS_Log.out
P3-INDOX-POST-ENCAPS_Log.progress.out
P3-INDOX-POST-ENCAPS_ReadsPerGene.out.tab
P3-INDOX-POST-ENCAPS_SJ.out.tab
P3-INDOX-PRE-ENCAPS_Log.final.out
P3-INDOX-PRE-ENCAPS_Log.out
P3-INDOX-PRE-ENCAPS_Log.progress.out
P3-INDOX-PRE-ENCAPS_ReadsPerGene.out.tab
P3-INDOX-PRE-ENCAPS_SJ.out.tab
```

**Comment by ewels** [comment]
Potentially a duplicate of https://github.com/ewels/MultiQC/issues/982

**Comment by agroppi** [comment]
Thanks.
I've looked at #982, but I don't understand what I have to do to correct this problem...

**Comment by ewels** [comment]
Wait until #982 is fixed 😄 

I haven't had time to look at it properly yet, but my guess is that STAR has updated the format of its log files, so I have to update the MultiQC code to be able to understand the new format.

If you or anyone else would like to have a go at writing the requisite code, you're welcome to open a pull-request..

Phil

**Comment by ewels** [comment]
Hi @agroppi,

Apologies - I was talking rubbish (not properly reading #982 or your issue). This is totally separate.

Hopefully your problem is covered by the docs here: https://multiqc.info/docs/#not-enough-samples-found

The clue is in the following debug messages:

```
[2019-07-04 17:58:45,049] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-CTRL-M17-T-AMB
[2019-07-04 17:58:45,139] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-CTRL-PRE-ENCAPS
[2019-07-04 17:58:45,228] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P1-INDOX-PRE-ENCAPS
[2019-07-04 17:58:45,377] multiqc.modules.star.star                          [DEBUG  ]  Duplicate ReadsPerGene sample name found! Overwriting: P2-CTRL-POST-ENCAPS
```

Does this help?

Phil

**Comment by agroppi** [comment]
Hi @ewels 

I have tried with -d and or -s with separate directories and or all log renamed in the same directory
But unfortunatly I had the same result : multiqc report show the 30 General Statistics but for the STAR Alignment scores and Gene counts Multiqc show only the plots for 15 samples.

**Comment by ewels** [comment]
Hi @agroppi,

I just opened the report that you attached and I think I now understand what's happening. When you say that you only have the plots for 15 samples, I guess that you're referring to the 15 labels here:

![image](https://user-images.githubusercontent.com/465550/60845829-c9fa9e00-a1dd-11e9-9284-ec0714bfa5aa.png)

However, you might notice that there are 30 _bars_ - the plotting library can't fit in all of the labels, so it's skipping every other sample.

If I expand the plot a little vertically, they appear:

![image](https://user-images.githubusercontent.com/465550/60845868-edbde400-a1dd-11e9-9a40-8d4c3aa653d5.png)

See the documentation about [interactive plots](https://multiqc.info/docs/#interactive-plots) and the grey bar to see how to expand plots like this.

Have I got this right this time?

Phil
